types/underscore/index.d.ts

```diff
@@ -6208,6 +6240,18 @@ declare module _ {
          **/
         matcher(): _Chain<T>;
 
+        /**
+         * Wrapped type `any`.
+         * @see _.get
+         **/
+        get(
+            path: string | string[]
+        ): _Chain<TypeOfCollection<V> | undefined>;
```

Comment:
Not sure yet what you should put here instead. Maybe `_ChainSingle`. I'll think about it.